dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/compilation/VisitorLKMM.java

```diff
@@ -69,8 +65,8 @@ public List<Event> visitLKMMCmpXchg(LKMMCmpXchg e) {
         loadFail.addTags(Tag.RMW);
         Load loadSuccess;
         return eventSequence(
-                newValueFunctionCall(havocRegister,nondetBoolFunction,List.of()),
-                newJump(expressions.makeBooleanCast(havocRegister), success),
+                EventFactory.Svcomp.newNonDetChoice(havocRegister),
```

Comment:
Probably it does not make much sense to keep having this as an svcomp event

Reply:
It doesn't make sense, but that is the current state for all our special events, including spinloops and loop bounds, so I kept it at that. In fact, some might argue that it does more closely belong to SVCOMP than the other two events, because it actually finds usage there.

Reply:
Yeah, all those events initially came from `VERIFIER_X` annotations and only later we end up using them for other purposes or having them automatically added. We might move them in the future, but this is no blocker for this PR.